ngx-fudis/projects/ngx-fudis/src/lib/components/dropdown-menu/dropdown-menu-item/dropdown-menu-item.component.html

```diff
@@ -1,54 +1,22 @@
 <button
-  *ngIf="!_parentComponent.multiselect"
   #dropdownItem
-  class="fudis-dropdown-menu-item fudis-dropdown-menu-item__focusable"
-  [attr.role]="'menuitem'"
+  class="fudis-dropdown-menu-item"
+  [class.fudis-dropdown-menu-item--disabled]="disabled || null"
+  [class.fudis-dropdown-menu-item__focusable]="!disabled"
   [attr.id]="_id"
```

Comment:
As a usability improvement I would add  tabindex="-1" here, which would make menu items unfocusable by tab key, while tab key would be preserved for moving out of menu. This way user doesn't have to tab through all menu items in order to exit while only tabing forward.

Reply:
Didn't know this was a good practice. Good to know, added it.